homeassistant/components/sensor/buienradar.py

```diff
@@ -198,6 +217,11 @@ def load_data(self, data):
                                            PRECIPITATION_FORECAST, STATIONNAME,
                                            TIMEFRAME)
 
+        # Check if we have a new measurement,
+        # otherwise we do not have to update the sensor
+        if self._measured == data.get(MEASURED):
```

Comment:
@balloob The check is done here, `self._measured` and `data.get(MEASURED)` are a `datetime` object. If they are the same (no new measurement loaded from buienradar) no new state is saved.

Reply:
Exactly. And because force update is set to True, Home Assistant will still write it to the state machine, which is wrong.

Reply:
I misunderstood, this is correct.